client/reader/controller.js

```diff
@@ -300,3 +300,22 @@ module.exports = {
 		);
 	}
 };
+
+export default exported;
```

Comment:
Yes, I think so - all instances of this module being imported was using named imports. 

I'll go ahead and remove this.